modules/windows-machineset-vsphere.adoc

```diff
@@ -67,7 +67,7 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 <2> Specify the Windows machine set name. The machine set name cannot be more than 9 characters long, due to the way machine names are generated in vSphere.
 <3> Configure the machine set as a Windows machine.
 <4> Configure the Windows node as a compute machine.
-<5> Specify the vSphere VM network to deploy the machine set to.
+<5> Specify the vSphere VM network to deploy the machine set to. This VM network must be where other Linux compute machines reside in the cluster.
 <6> Specify the full path of the Windows vSphere VM template to use, such as `/Datacenter/vm/ocp4-llplx/windows-golden-image`. The name must be unique.
 +
 [IMPORTANT]
```

Comment:
> <6> Specify the full path of the Windows vSphere VM template to use, such as `/Datacenter/vm/ocp4-llplx/windows-golden-image`. The name must be unique.

This is referring to the VM template , we just created in above step: [Convert the Windows VM to a template in vCenter](https://github.com/openshift/openshift-docs/pull/35728/files#diff-a9972807dbbd53e5423f681de126b7ec7a6e26d87bea6a6232b1e8d4e192a686R147)

Not sure how it relates to RHCOS (`{op-system}`) machines. Should we remove this [IMPORTANT] note? 

cc: @openshift/openshift-team-windows-containers

Reply:
The `{op-system}` needs to be removed or reference Windows but rest of the note needs to stay given users should not specify the original Windows VM template either

Reply:
Yeah, that was a copy/paste error from the original vSphere machine sets docs. Fixed.